File: resilience_eval/benchmarks/java/ipa_cp_1.java
public class interprocedural_constant_propagation_1 {
	int ipa_cp_inlined(int x, int y, int c){
		return x+y+c;
	}
	int interprocedural_constant_propagation_1(int input0, int input1){
		return ipa_cp_inlined(input0, input1, 1);
	}
}

File: resilience_eval/benchmarks/scala/ipa_cp_1.scala
class interprocedural_constant_propagation_1 {
	def ipa_cp_inlined(x:Int, y:Int, c:Int): Int = {
		return x+y+c;
	}
	def interprocedural_constant_propagation_1(input0:Int, input1:Int): Int = {
		return ipa_cp_inlined(input0, input1, 1);
	}
}
